src/main/fc/cli.c

```diff
@@ -3193,6 +3196,18 @@ static void cliStatus(char *cmdline)
         hardwareSensorStatusNames[getHwGPSStatus()]
     );
 
+#ifdef USE_ESC_SENSOR
+    uint8_t motorCount = getMotorCount();
+    cliPrintLinef("ESC Temperature(s): Motor Count=%d",motorCount);
```

Comment:
@Christoph-Hofmann this is a very cool addition. I always struggle to verify if ESC telemetry works.
Let's however, improve it and display this line onli if 
`if (STATE(ESC_SENSOR_ENABLED) && getMotorCount() > 0) {`